SecretSauce/src/main/java/com/byoutline/secretsauce/SecretSauceSettings.kt

```diff
@@ -52,10 +52,18 @@ object SecretSauceSettings {
         this.viewModelFactoryProvider = viewModelFactoryProvider
         this.useFragmentViewModelProvider = useFragmentViewModelProvider
         if (setFastJodaTimeZoneProvider) {
+            setFastJodaTimeZoneProvider()
+        }
+    }
+
+    private fun setFastJodaTimeZoneProvider() {
+        try {
+            Class.forName("org.joda.time.DateTimeZone")
```

Comment:
This will throw if there is no `org.joda.time.DateTimeZone` in classpath.
This check is not strictly necessary, as settings System.property usually is not expensive and would probably be ignored, but current behaviour seems cleaner.
After all if there is no Joda dependency in classpath attempting to import JdkBasedTimeZoneProvider will crash.